config/routes.rb

```diff
@@ -1,21 +1,30 @@
 Rails.application.routes.draw do
+
   devise_for :users
   root "static_pages#home"
   get "/home", to: "static_pages#home", as: "home"
-  get "/learn-more", to: "static_pages#learn_more", as: "learn_more"
-  get "/tools-resources", to: "static_pages#tools_resources", as: "tools_resources"
-  get "/techhire-locations", to: "static_pages#techhire_locations", as: "techhire_locations"
+  get "/learn_more", to: "static_pages#learn_more", as: "learn_more"
+  get "/tools_resources", to: "static_pages#tools_resources", as: "tools_resources"
+  get "/techhire_locations", to: "static_pages#techhire_locations", as: "techhire_locations"
   get '/admin', to: 'sessions#new', as: 'admin/login'
   post '/admin', to: 'sessions#create'
   get '/download', to: "static_pages#download", as: "download"
 
   namespace :admin do
     resources :dashboard, only: [:index]
+    resources :contents
+    resources :videos
+    resources :pdfs
+    resources :techhire_locations
     resources :companies, only: [:show, :destroy, :update, :index] do
       resources :notes, only: [:index, :create, :destroy]
     end
-    resources :pdfs, only: [:create, :new]
-    resource :tools_resources, only: [:edit]
+    namespace :static_pages do
```

Comment:
These bottom routes that are namespaced under static_pages are for changing content. When an admin goes to admin/static_pages/learn_more, he can edit the actual page. The resources supported here are edit/update. On this page, they can swap out content/video/pdfs for ones in the database.